t/perf/run

```diff
@@ -74,7 +74,7 @@ set_git_test_installed () {
 	mydir=$1
 
 	mydir_abs=$(cd $mydir && pwd)
-	mydir_abs_wrappers="$mydir_abs_wrappers/bin-wrappers"
+	mydir_abs_wrappers="$mydir_abs/bin-wrappers"
```

Comment:
Just an FYI: I sent this upstream via gitgitgadget/git#1044, since it is so simple.

See https://lore.kernel.org/git/pull.1044.git.1632239172735.gitgitgadget@gmail.com/T/#u for the discussion.

Reply:
In that case, should I remove it from this branch? Or will it disappear once `vfs-2.33.0` is rebased?

Reply:
It will disappear during the rebase onto `v2.34.0` (assuming it gets in quickly).